politeiawww/www.go

```diff
@@ -631,6 +632,23 @@ func _main() error {
 				return fmt.Errorf("build cache: %v", err)
 			}
 		}
+		if p.cfg.GithubAPIToken != "" {
+			if p.cfg.CodeStatOrganization != "" {
```

Comment:
Why do you use nested if statements instead of `if p.cfg.GithubAPIToken != "" && p.cfg.CodeStatOrganization != "" {}`?